flymake-eslint.el

```diff
@@ -68,6 +69,13 @@ directory of the linting process."
   :type 'string
   :group 'flymake-eslint)
 
+(defcustom flymake-eslint-prefer-json-diagnostics nil
+  "Try to use the JSON diagnostic format when runnin ESLint.
```

Comment:
nitpick: typo ("runnin" -> "running")

Reply:
Thanks! I ran Jinx to check for other spelling mistakes.